config/certmanager/kustomization.yaml

```diff
@@ -1,5 +1,6 @@
 resources:
 - certificate.yaml
+- certificate-metrics.yaml
```

Comment:
does it make sense to enable this by default?

Reply:
By default cert-manager is disabled. If we enable this by default, I think we would get `Certificate` resource not found error?.

Reply:
Ok makes sense, let's secure metrics regardless of prometheus.